packages/ibm-products/src/components/FilterPanel/FilterPanelCheckbox/FilterPanelCheckbox.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { Checkbox } from '@carbon/react';
+import { FilterPanelLabel } from '../FilterPanelLabel';
+
+import { getDevtoolsProps } from '../../../global/js/utils/devtools';
+import { pkg } from '../../../settings';
+
+// The block part of our conventional BEM class names (blockClass__E--M).
+const blockClass = `${pkg.prefix}--filter-panel-checkbox`;
+const componentName = 'FilterPanelCheckbox';
+
+/**
+ * Provides a checkbox, label, and count.
+ */
+export let FilterPanelCheckbox = React.forwardRef(
+  (
+    { className, count, labelText, labelTitle, setCountAriaLabel, ...rest },
+    ref
+  ) => {
+    return (
+      <Checkbox
+        {...rest}
+        className={cx(blockClass, className)}
+        labelText={
+          <FilterPanelLabel
+            count={count}
+            setCountAriaLabel={setCountAriaLabel}
+            labelTitle={labelTitle || labelText}
```

Comment:
since the underlying `title` attribute is unnecessary we should be able to just stick to one prop value here

Reply:
Done.